catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/ShareableMetacardPolicyExtensionTest.java

```diff
@@ -36,8 +35,8 @@
 
 public class ShareableMetacardPolicyExtensionTest {
```

Comment:
❗️ This needs a lot more unit tests for the different match one and match all cases.

Reply:
Agreed. This looks correct to me at this point, but I want to see a bunch more unit tests before I give it a 👍